pacehrh/R/pace_save_results.R

```diff
@@ -293,7 +293,7 @@ SaveSuiteResults <- function(results, filepath, scenario, run){
   # row names in a column called "rn")
   DTt <- data.table::as.data.table(mt, keep.rownames = TRUE)
   DT <- data.table::melt(DTt, c("rn"))
-  names(DT) <- c("Task_ID", "Year", "Service_time")
+  names(DT) <- c("Year", "Task_ID", "Service_time")
```

Comment:
Wow. Well spotted! It's been ages since we've cared about results without seasonality.